release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1636,6 +1642,14 @@ sourceStrategy:
 +
 To opt out of using the VHD feature, remove the `fstype` parameter from your storage class. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2080449[*BZ#2080449*])
 
+[discrete]
+[id="ocp-4-12-windows-containers-bug-fixes"]
+==== Windows containers
+
+* Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for containerd on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the Windows Machine Config Operator (WMCO) can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (link:https://issues.redhat.com/browse/OCPBUGS-3573[*OCPBUGS-3573*])
```

Comment:
WINC is an asynchronous release and the release that contains this fix will not release with 4.12, likely on 1/30/23. Can you hide this release note section and I will un-hide when WINC releases?

Reply:
Or we can leave it in the WINC release notes only. That might be a more technically accurate approach.

Reply:
Yeah I think it would be good to have these in the WINC rel notes. Thanks!